packages/common/src/core/types/IHubContent.ts

```diff
@@ -19,7 +19,7 @@ export enum PublisherSource {
  * Data model for content
  */
 export interface IHubContent
-  extends IHubItemEntity,
+  extends Omit<IHubItemEntity, "schemaVersion">,
```

Comment:
This allows `schemaVersion` to be added to `IHubItemEntity` but to not be part of `IHubContent`